assets/app/view/game/entities.rb

```diff
@@ -26,9 +26,9 @@ def render
 
         player_owned, bank_owned = (@game.corporations + @game.minors)
           .reject(&:closed?)
-          .sort_by(&:name)
           .partition(&:owner)
-        player_owned = player_owned.group_by(&:owner)
+        player_owned = player_owned.sort_by(&:name).group_by(&:owner)
+        bank_owned.sort_by!(&@game.bank_sort)
```

Comment:
i don't like &@game.bank_sort

Reply:
Have you a better idea?

Reply:
no haha

Reply:
not having this custom sort is one...

Reply:
The order in which the corporations will be opened is otherwise inaccessible during the private auction. I could log the order but that's pretty lame. Also, supporting these custom sorts can also be useful in games with e.g. minors (say, 1846, in which at the start of the game the minors are mixed in with the majors alphabetically. I might argue it'd be better if they were grouped together instead.)

Reply:
Why not change the definition of '<=>' for unfloated corps?

Reply:
if that solution is generic that'd be great

Reply:
I wish that could work but I don't think it will. Corps have no knowledge of where they are in the opening order (that information exists only in the ordering of the `@corporations` array in `Game`). For that to work I think I'd need to add `@game` as a field in `Corporation`, and `<=>` would have to walk through the `@game.corporations` array to find its index and `other`'s index. Is that better than this?

Reply:
Another way would be to store the corp's index in `Game`'s `@corporations` array as a separate field, and update it every time the order changes, but that's a lot more code in many more places than what I have here.

Reply:
ok can we instead change it to be

game.nice_method_name_to_sort_for_bank(corporations)

Reply:
done

Reply:
But given a corporation A and a corporation B, they can't figure out by themselves which one starts before?

Reply:
Not without doing one of the things I listed above.

Reply:
I don't know 1849. I assumed it was easier.